scripts/automated-patch-releases.mermaid

```diff
@@ -0,0 +1,11 @@
+---
+title: How CRI-O Automated Patch Releases Work
+---
+
+flowchart TD
+  A[Start: Patch Release Workflow] --> B[Create Version Bump Intent PRs]
```

Comment:
Name the actual workflows here. We need a reference: https://github.com/cri-o/cri-o/actions/workflows/patch-release.yml